graphql-dgs-subscriptions-sse/src/main/kotlin/com/netflix/graphql/dgs/subscriptions/sse/DgsSSESubscriptionHandler.kt

```diff
@@ -139,7 +140,7 @@ open class DgsSSESubscriptionHandler(open val dgsQueryExecutor: DgsQueryExecutor
     }
 
     companion object {
-        private val mapper = jacksonObjectMapper()
+        private val mapper = jacksonObjectMapper().registerModule(Jdk8Module())
```

Comment:
Maybe we should just use the `Jackson2ObjectMapperBuilder` from Spring, like we do elsewhere. It will automatically register these well-known modules if they're there on the classpath.

Reply:
[Example](https://github.com/Netflix/dgs-framework/blob/872b0e76b8130d57766584d2ba210728695fbd69/graphql-dgs-client/src/main/kotlin/com/netflix/graphql/dgs/client/GraphQLClients.kt#L29), except the usage here would be even simpler since the KotlinModule isn't being customized.

Reply:
Thank you! Yes, let's maintain consistency. Updated to use Jackson2ObjectMapperBuilder